content/1.3-migration.md

```diff
@@ -0,0 +1,73 @@
+## Migrating to use imports everywhere
+
+Never use a symbol without importing it (i.e. no more psuedo-globals!).
+
+In general from core Meteor libraries your code should remain unchanged, and you just add some lines like:
+
+```js
+import {Meteor} from 'meteor/meteor';
+import {ReactiveVar} from 'meteor/reactive-var';
+```
+
+With the following exceptions
```

Comment:
Why are these exceptions? Maybe something like "These are the places where packages used to attach stuff to a different namespace, so we created a new one so you know which package it's coming from" or something.

Speaking of which, we should remind package authors to start doing this as well, like publish-composite